### DISCUSSION TITLE
Icons on Title Bar Blank

### DISCUSSION THREAD

**Discussion by vipul128** [elax46/custom-brand-icons#790] [Q&A]
I've had an issue for a while where I'm using custom-brand-icons on the titlebar of my dashboard but most of the time they show up blank.  I found another thread that said to modify the configuration.yaml file and I've done that and it still has not fixed my issue.  Hoping someone here can help:

My configuration.yaml file:
```
frontend:
  extra_module_url: /homeassistant/www/community/custom-brand-icons/custom-brand-icons.js
``` 

Screenshot (blanks are where the icons should be):
<img width="949" alt="Screenshot 2024-10-02 at 8 11 17 PM" src="https://github.com/user-attachments/assets/cc3b76ef-3880-4590-8ace-d83d8e1131e0">

Thanks in advance!

**Comment by elax46** [comment]
Hi we have been aware of this problem for some time but it does not depend on CBI but on the cache policies of home assistant and browsers have you verified that the `extra_module_url `link to the js file is the correct path with respect to your CBI installation?

**Reply by vipul128** [reply]
Yes, the URL is a link to the .js file.  See screenshot below from file editor:

<img width="599" alt="image" src="https://github.com/user-attachments/assets/50df91af-2350-4337-8552-f16862519a88">
